src/xpk/core/kueue.py

```diff
@@ -140,6 +141,43 @@
 """
 
 
+def prepare_kueuectl(args: Namespace) -> None:
```

Comment:
I think this method is a leftover from before when you were installing kueuectl because now it doesn't seem to prepare anything and is basically just a wrapper for `verify_.." method. Maybe we can combine them?

Reply:
they are combined but the name is still `prepare..` . I think `verify...` is more descriptive.